eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -19,14 +19,16 @@ public class ConfigurationStore implements IConfigurationStore {
   private static final String TAG = logTag(ConfigurationStore.class);
   private final ConfigCacheFile cacheFile;
   private final Object cacheLock = new Object();
-  private Configuration configuration;
+
+  // default to an empty config
+  private volatile Configuration configuration = Configuration.emptyConfig();
```

Comment:
👍

Reply:
why volatile? this can be changed from multiple threads?